client/README.md

```diff
@@ -98,16 +102,32 @@ Every public address in Orbs platform has the form of {Network ID, Address Schem
   * Source: [https://github.com/google/googletest.git](https://github.com/google/googletest.git)
   * Version: `release-1.8.0` git tag.
 * Libgpg-error (`libgpg_error`): Libgpg-error is a small library that originally defined common error values for all GnuPG components.
-  * Source: [https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.27.tar.bz2](https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.27.tar.bz2)
-  * Version: 1.27.
+  * Source: [https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.28.tar.bz2](https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.28.tar.bz2)
+  * Version: 1.28.
 * Libgcrypt (`libgcrypt`): Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG.
   * Source: [https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.2.tar.bz2](https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.2.tar.bz2)
   * Version: 1.8.2.
 
 ## Prerequisites
 
+### General
+
 1. [CMake](https://cmake.org)
 
+### iOS
+
+1. [Xcode](https://developer.apple.com/xcode/)
+2. iPhone Simulator (64bit)
+
+### Android
+
+1. [Android Studio](https://developer.android.com/studio/index.html) (3.1 or higher)
+2. [Android SDK](https://developer.android.com/studio/intro/update.html)
```

Comment:
The Layman's guidelines are to install all these (including `Android NDK`) via the `SDK Manager`. Alternatively, add a shell script that installs them via the `sdkmanager` cmd tool

Reply:
More prerequisites: JDK 1.7+, Gradle

Reply:
I'll update the README.md